bot/seasons/evergreen/game.py

```diff
@@ -194,9 +197,23 @@ def cog_unload(self) -> None:
         try:
             games = await self.get_games_list(amount, self.genres[genre], offset=random.randint(0, 150))
         except KeyError:
-            possibilities = "`, `".join(difflib.get_close_matches(genre, self.genres))
-            await ctx.send(f"Invalid genre `{genre}`. {f'Maybe you meant `{possibilities}`?' if possibilities else ''}")
-            return
+            possibilities = await self.get_best_results(genre)
+            possibility = None
+            # Check is there any possibility that ratio is higher than 0.60
+            for p in possibilities:
+                if p[0] >= 0.60:
+                    possibility = await self.get_games_list(amount, self.genres[p[1]], offset=random.randint(0, 150))
+                    genre = p[1]
+                    break
```

Comment:
This part can use extra logics check, checking for the highest score might not always work.

However, currently it is working pretty well with all the test cases I provided, so you can argue otherwise that being practical is better in this case. Still, I am keen on seeing a suggestion on this part,

Reply:
Maybe one thing that this should check is the first letter?

Reply:
It will be more complicated than that, for example when you do `.games and` you will actually get `Point and click` and you will not see `hack-and-slash` because the check grab the first result already.

Granted that no one should ever search for a genre called `and` this is something we can do to to improve user's experience.

An example of what we can do, is that if all returned results have scores greater than 0.6 we can list all of them instead, like when searching for tags and there are multiple matching suggestions.

Reply:
I added now this check. Is now better?

Reply:
It does sound better in theory, I suggest modifying `get_best_results()` to accept an optional argument for score, then you only have to check for the length of the list it returns, that'll simplify the checks outside!

Reply:
I don't see reason for the additional argument, so I hardcoded it, due this is used only in one place. I also simplified checking in `.games` command side.

Reply:
That's true.

In your new code, 
```py
display_possibilities = "`, `".join(p[1] for p in possibilities if p[0] >= 0.40)
```
Will practically be the same as
```py
display_possibilities  = "`, `".join(p[1] for p in possibilities)
```
Because the function only returns genres with ratio >= 0.6 - the check for >= 0.4 is redundant.

Reply:
Okay, fixed it.

Reply:
Working great!